dev/TeachingTip/TeachingTip.h

```diff
@@ -125,6 +125,8 @@ class TeachingTip :
     void OnTargetLayoutUpdated(const winrt::IInspectable&, const winrt::IInspectable&);
     void OnTargetLoaded(const winrt::IInspectable&, const winrt::IInspectable&);
     void RepositionPopup();
+    void ClosePopupOnUnloadEvent(winrt::IInspectable const&, winrt::RoutedEventArgs const& e);
+    winrt::event_token m_TargetUnloadToken;
```

Comment:
>winrt::event_token m_TargetUnloadToken; [](start = 4, length = 39)

We prefer to use the auto revoker pattern when registering to events. The code as written has the potential to leak as we are not detaching this event when we clean up teaching tip, which means we could accidently keep a reference to the target preventing it's clean up.

Search the control for the term revoker and you should be able to get a handle on the pattern, it handles the detaching of the events during clean up for you by having code in its destructor to tear down connection.

Reply:
I switched to using a Unloaded_revoker. I didn't knew that this existed 😅 
Thank you for pointing that out!